sfge/src/main/java/com/salesforce/rules/AvoidDatabaseOperationInLoop.java

```diff
@@ -15,6 +15,9 @@ public final class AvoidDatabaseOperationInLoop extends AbstractPathTraversalRul
     private static final ImmutableSet<ApexPathSource.Type> SOURCE_TYPES =
             ImmutableSet.copyOf(ApexPathSource.Type.values());
 
+    private static final String URL =
```

Comment:
do we have config system for string literals?

Reply:
are there any integration tests that can hit these URLs in tests or test only URLs?

Reply:
I don't think so. We could look into adding some in the future, though.